### PR TITLE
feat: adding an initial set of sample templates

### PR DESCRIPTION
**Pull request by ddneilson**
We are providing some sample Job/Environment Templates that users can reference to demonstrate the sorts of things that can be done with Open Job Description. This seeds an initial set of templates.

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,21 +3,21 @@
 Thank you for your interest in contributing to our project. Whether it's a bug report, new feature, correction, or additional
 documentation, we greatly value feedback and contributions from our community.
 
-Open Job Description is under active development. We're releasing this because we'd like your feedback and participation
```
Comment (shidarin) [inline CONTRIBUTING.md]: Would it be possible to back out the style changes and submit as a separate PR so it can be a sep commit?

File: samples/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Open Job Description Sample Templates
+
+This directory contains a set of sample templates for the Open Job Description standard.
+The aim with these is to provide a small sampling of just some of the things that you
+can do with Open Job Description, and a reference set of demonstrations of template
+concepts that you can incorporate in to your own templates.
+
+All samples are provided AS-IS. We strongly recommend that you
```
Comment (shidarin) [inline samples/README.md]: We strongly recommend that you test all samples thoroughly before use, and customize the samples as required by your individual needs.

**Review by shidarin — CHANGES_REQUESTED** [review]
Half finished sentence.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ddneilson) [reply CONTRIBUTING.md]: Sure. I'll add the spaces back in.

File: README.md [hunk 3]
```diff
@@ -7,33 +7,57 @@ and to drive updates from the community.
...
-release render farm plugins for their own applications despite being the most 
-qualified to do so, as the render farm landscape was fractured and releasing 
-multiple plugins for different render farm solutions would quickly siphon all 
-their development capacity away from improving their software. A standard way of 
-defining render jobs, or really any type of work suited for completion on a 
-render farm, makes tools and plugins portable and increases interoperability. 
-
-For deeper insight into our thought process and goals, we recommend [this Academy Software Foundation talk](https://www.youtube.com/watch?v=3AM3L6P-cAw&list=PL9dZxafYCWmxDFGc2CEq4SgCkZWKYW1m5&index=14)
-by Pauline Koh, Senior Product Manager at Amazon Web Services, titled 
+Open Job Description is a flexible open specification for defining render
```
Comment (ddneilson) [inline README.md:24]: I don't know what it thinks that I changed here.

**Review by shidarin — APPROVED** [review]
